.github/workflows/unit-tests.yml

```diff
@@ -113,16 +113,13 @@ jobs:
         with:
           path: ${{ github.run_id }}
 
-      - name: Run tests
+      - name: Run all tests
         env:
           BIONEMO_DATA_SOURCE: ngc
-        run: ./ci/scripts/run_pytest.sh --no-nbval --skip-slow
-
-      - name: Run notebook tests
-        if: ${{ contains(github.event.pull_request.labels.*.name, 'INCLUDE_NOTEBOOKS_TESTS') }}
-        env:
-          BIONEMO_DATA_SOURCE: ngc
-        run: pytest --nbval-lax -p no:python docs/ sub-packages/
+        run: >
+          ./ci/scripts/run_pytest.sh
+          ${{ !contains(github.event.pull_request.labels.*.name, 'INCLUDE_NOTEBOOKS_TESTS') && '--no-nbval' || '' }}
+          ${{ !contains(github.event.pull_request.labels.*.name, 'test/L1') && '--skip-slow' || '' }}
```

Comment:
Do we want to go with the L1 etc syntax? I feel like we need a long doc describing what L1 is when we can have something like `INCLUDE_SLOW_TESTS` wouldn't require a tutorial on what L1/L2/... are. When I see L1 I think L1 cache on CPUs. I don't know what L1 cache on CPUs is, but I remember that CPUs used to advertise something called that.